docs/usage/code_snippets.rst

```diff
@@ -11,22 +11,24 @@ DWD (German Weather Service)
 Historical Weather Observations
 -------------------------------
 
-Get available parameters for daily historical data:
+Get available parameters for daily historical data of DWD:
 
 .. ipython:: python
 
-    from wetterdienst.dwd.observations import DwdObservationRequest, DwdObservationResolution, DwdObservationPeriod
+    from wetterdienst import Wetterdienst, Resolution, Period
 
-    observations_meta = DwdObservationRequest.discover(
-        resolution=DwdObservationResolution.DAILY,
+    API = Wetterdienst("dwd", "observation")
```

Comment:
Can we use named parameters (keyword args) here and at all other occurrences like this?